maven-scm-providers/maven-scm-provider-hg/src/test/java/org/apache/maven/scm/provider/hg/HgRepoUtils.java

```diff
@@ -35,9 +35,11 @@
  *
  * @author <a href="mailto:thurner.rupert@ymono.net">thurner rupert</a>
  */
-public class HgRepoUtils
+public abstract class HgRepoUtils
```

Comment:
Why? This class even extends `ScmTestCase` for whatsoever reason and has no descendants.

Reply:
This failed in the previous testing framework. It no longer does so I have remove this change.